### PR TITLE
Garage Interns: Adding New Sample Queries

### PR DESCRIPTION
**Pull request by lhao03**
New queries to add to support more Microsoft Teams APIs.

### CODE REVIEW

File: sample-queries/sample-queries.json [hunk 1]
```diff
@@ -270,7 +270,7 @@
         "humanName": "get a user's presence ",
         "requestUrl": "/beta/users/{user-id}/presence",
         "docLink": "https://docs.microsoft.com/en-us/graph/api/cloudcommunications-getpresencesbyuserid?view=graph-rest-beta&tabs=http",
-        "tip": "This query requires a user id and Presence.Read.All permissions. To find user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
+        "tip": "This query requires a user id and Presence.Read.All permissions. To find a user id, you can run:  GET https://graph.microsoft.com/v1.0/users",
```
Comment (Lauragra) [inline sample-queries/sample-queries.json:273]: Revise as: "This query requires a user ID and the Presence.Read.All permission. Use the following call to get a user ID:"

File: sample-queries/sample-queries.json [hunk 2]
```diff
@@ -1435,6 +1451,23 @@
...
+        "requestUrl": "/v1.0/chats",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-post?view=graph-rest-1.0&tabs=http",
+        "headers": [
+            {
+                "name": "Content-Type",
+                "value": "application/json"
+            }
+        ],
+        "postBody": "{\r\n    \"chatType\": \"oneOnOne\",\r\n    \"members\": [\r\n    {\r\n    \"@odata.type\": \"#microsoft.graph.aadUserConversationMember\",\r\n    \"roles\": [\"owner\"],\r\n    \"user@odata.bind\": \"https://graph.microsoft.com/v1.0/users('{your-user-id}')\"\r\n    },\r\n    {\r\n    \"@odata.type\": \"#microsoft.graph.aadUserConversationMember\",\r\n    \"roles\": [\"owner\"],\r\n    \"user@odata.bind\": \"https://graph.microsoft.com/v1.0/users('{user-id}')\"\r\n    }\r\n    ]\r\n    }",
+        "tip": "This query requires your user id and another user's id.  To find your user id & another user's id, you can run: 1) GET https://graph.microsoft.com/v1.0/me/ 2) GET https://graph.microsoft.com/v1.0/users",
```
Comment (Lauragra) [inline sample-queries/sample-queries.json:1468]: Revise as: "This query requires your user ID and another user's ID. Use the following calls to get the user IDs:"

File: sample-queries/sample-queries.json [hunk 3]
```diff
@@ -1474,6 +1507,35 @@
...
+        "skipTest": false
+    },
+    {
+        "id": "25344aeb-872a-467d-b202-bdc3b21470b1",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "member in a chat",
+        "requestUrl": "/beta/chats/{chat-id}/members/{membership-id}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get-members?view=graph-rest-beta",
+        "tip": "This query requires a chat id and a membership id. To find the corresponding ids, run 1) GET https://graph.microsoft.com/beta/me/chats/ and 2) GET https://graph.microsoft.com/beta/me/chats/{chat-id}/members/",
```
Comment (Lauragra) [inline sample-queries/sample-queries.json:1537]: Revise as: "This query requires a chat ID and a membership ID. Use the following calls to get the IDs: "

File: sample-queries/sample-queries.json [hunk 4]
```diff
@@ -1474,6 +1507,35 @@
...
+        "skipTest": false
+    },
+    {
+        "id": "6d1a6a27-0400-425f-a05e-594669e8fbc1",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "list members of a chat",
+        "requestUrl": "/beta/chats/{chat-id}/members",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-list-members?view=graph-rest-beta",
+        "tip": "This query requires a chat id. To find a list of chats and their corresponding ids, run GET https://graph.microsoft.com/beta/me/chats/",
```
Comment (Lauragra) [inline sample-queries/sample-queries.json:1527]: Revise this one as: "This query requires a chat ID. Use the following call to find a list of chats and their corresponding IDs: "

File: sample-queries/sample-queries.json [hunk 5]
```diff
@@ -1474,6 +1507,35 @@
...
+        "skipTest": false
+    },
+    {
+        "id": "25344aeb-872a-467d-b202-bdc3b21470b1",
+        "category": "Microsoft Teams (beta)",
+        "method": "GET",
+        "humanName": "member in a chat",
+        "requestUrl": "/beta/chats/{chat-id}/members/{membership-id}",
+        "docLink": "https://docs.microsoft.com/en-us/graph/api/chat-get-members?view=graph-rest-beta",
+        "tip": "This query requires a chat ID and a membership ID. Use the following calls to get the IDs: run 1) GET https://graph.microsoft.com/beta/me/chats/ and 2) GET https://graph.microsoft.com/beta/me/chats/{chat-id}/members/",
```
Comment (Lauragra) [inline sample-queries/sample-queries.json:1537]: In this string, remove the word "run" after the colon.

**Review by irvinesunday — APPROVED** [review]
👍🏽

**Review by MaggieKimani1 — APPROVED** [review]

**Comment by lhao03** [comment]
Thanks for approving @irvinesunday and @MaggieKimani1! Since I don't have permissions to merge the PR, would you be able to do that if the PR is ready? Thanks 😊!

**Pull request merged**